eradiate/scenes/measure/_distant.py

```diff
@@ -529,9 +529,9 @@ def _postprocessing_add_illumination(
         self, ds: xr.Dataset, illumination: DirectionalIllumination
     ) -> xr.Dataset:
         # Collect illumination angular data
-        saa = illumination.azimuth.m_as(ureg.rad)
-        sza = illumination.zenith.m_as(ureg.rad)
-        cos_sza = np.cos(sza)
+        saa = illumination.azimuth.m_as(ureg.deg)
+        sza = illumination.zenith.m_as(ureg.deg)
+        cos_sza = np.cos(illumination.zenith.m_as(ureg.rad))
```

Comment:
For simplicity and to avoid unnecessary Pint conversions, I'd suggest:
```suggestion
        cos_sza = np.cos(np.deg2rad(sza))
```
[Docs here](https://numpy.org/doc/stable/reference/generated/numpy.deg2rad.html)